### DISCUSSION TITLE
Airsim binary to sln file

### DISCUSSION THREAD

**Discussion by kwanyongcho7** [microsoft/AirSim#3796] [Support Q&A]
Hi, I downloaded ZhangJiajie.exe to play my HITL.

But, it is too high quality for my computer and I think I have to play it on ue4.

so I tried to generate visual studio solution but it is not working.. 

![image](https://user-images.githubusercontent.com/22126137/122163750-a0aa1880-ceb0-11eb-8168-3dcf1bc47b19.png)

is there any way I can make it as sln file?

**Comment by jonyMarino** [comment]
Hi @kwanyongcho7! there is no way to get a .sln from binaries.  Try to use a lower resolution.
You can execute a command like this on your command line:
ZhangJiajie.exe -ResX=640 -ResY=480 -windowed
